incuna_auth/locale/en_IE/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,213 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```

Comment:
Can you remove the fuzzy and optionally everything above? And can the same be done with the others po files?